### PR TITLE
Made the difference between scale-up timeout and cluster set-up timeout explicit.

### PR DESCRIPTION
**Pull request by wasylkowski**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wasylkowski** [comment]
CC @MaciekPytel

File: test/e2e/autoscaling/cluster_autoscaler_scalability.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ import (
 )
 
 const (
+	clusterSetUpTimeout   = 5 * time.Minute
```
Comment (MaciekPytel) [inline test/e2e/autoscaling/cluster_autoscaler_scalability.go]: I'm not sure I like this name all that much. We use it in test set-up, but what we do is really just run some pods that we believe will fit on the existing nodes and wait for them to get scheduled. So I'd rather call it something like memoryReservationScheduleTimeout. I don't have a very strong opinion on this though, so if you think your name is better I'm ok to proceed with it.

Comment (wasylkowski) [inline test/e2e/autoscaling/cluster_autoscaler_scalability.go]: Renamed to memoryReservationTimeout (shorter and just as meaningful)

**Comment by MaciekPytel** [comment]
@kubernetes/test-infra-maintainers @kubernetes/sig-testing-bugs It seems there is some issue with bot repeatedly ignoring `/approve no-issue`. Both me and @wasylkowski are approvers for the files covered in this change, yet the bot still claims it's not approved.

**Comment by wojtek-t** [comment]
Strange. Let me try.

/approve no-issue

**Comment by wojtek-t** [comment]
Even more strange - that worked for me

**Pull request merged**